new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,6 +49,46 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info("Got a new namespace called {0}", namespace);
 
+    // create persistent volume and persistent volume claim
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+
+    V1ObjectMeta pvcmetadata = new V1ObjectMetaBuilder()
```

Comment:
No need for builder or variable here.  Move into position on line 70.

Reply:
Just create a variable for the name since you use that below.

Reply:
corrected it.